002-quarkus-all-extensions/pom.xml

```diff
@@ -104,15 +103,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-metrics</artifactId>
         </dependency>
-        <!--        <dependency>  needs running Keycloak server -->
-        <!--            <groupId>io.quarkus</groupId>-->
-        <!--            <artifactId>quarkus-keycloak-authorization</artifactId>-->
-        <!--        </dependency>-->
-        <!--  NATIVE mode issue -->
-        <!-- <dependency>
-          <groupId>io.quarkus</groupId>
-          <artifactId>quarkus-hibernate-search-orm-elasticsearch</artifactId>
-        </dependency> -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-keycloak-authorization</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-oidc</artifactId>
```

Comment:
You do enable these 2 extensions (oidc, kc authz) and add `KeycloakTestResource` for them. However you don't have any tests calling them. The whole point I take it is that they are present and enabled. That is OK, but I think it's bit overkill to add dedicated Keycloak resource, start it when we don't use it. You set:

```
quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/test-realm
quarkus.oidc.client-id=quarkus-app
```

I think it would test about same because OIDC would provide auth server itself if you didn't set these properties.

Anyway, that is just an opinion, you can keep it. You can say it's healthy to test dedicated resource server instead of Dev Svc.

Reply:
Oidc doesn't run server by itself and will require set properties. I want to leave test with resource. It will cost our CI nothing as It is not running here because it requires a lot of RAM.

Reply:
> Oidc doesn't run server by itself and will require set properties. 

not true


>  I want to leave test with resource. 

that's ok

> It will cost our CI nothing as It is not running here because it requires a lot of RAM.

I don't get it. How starting a container cost nothing and if it is not running, why are you adding it?

Reply:
We don't run 002 module in CI (reason above), you can run this module locally

Reply:
:-D

Reply:
Who will ever do that? Let's drop that module.

Reply:
Anyway, in that case I don't care, I'll close this comment.